Atomic/atomic.py

```diff
@@ -806,10 +811,14 @@ def install(self):
         if self._container_exists(self.name):
             raise ValueError("A container '%s' is already present" % self.name)
 
-        if self.system:
+        if self.user:
+            if not util.is_user_mode():
+                raise ValueError("--user does not work for privileged user")
+            return self.syscontainers.install_user_container(self.image, self.name)
+        elif self.system:
             return self.syscontainers.install_system_container(self.image, self.name)
         elif self.args.setvalues:
-            raise ValueError("--set is valid only when used with --system")
+            raise ValueError("--set is valid only when used with --system or --user")
 
```

Comment:
Same we should do this in the parser code.  Can we check at that point if we are doing --system